### DISCUSSION TITLE
LRc 13.5 and MIDI2LR 6.1.0.0 - Not connected to Lightroom - ??

### DISCUSSION THREAD

**Discussion by Drounding** [rsjaffe/MIDI2LR#792] [Problems]
I've been using LRc with MIDI2LR for years. Today I upgraded LRc to 13.5 and MIDI2LR (an older 5.? release) won't connect to LRc.
I upgraded MIDI2LR to the latest 6.1.0.0 but still won't connect. I have some wedding shots I need to work on, can you please help me get it working again?

**Comment by antzpantz** [comment]
That's quite the upgrade. Have you tried going back to the older version?
Did you need any of the functions of 6.1.0.0? If you do, I'd recommend
backing up your MIDI2LR settings (there are guides available) and
deleting/uninstalling MIDI2LR completely and reinstalling it.

**Reply by Drounding** [reply]
I can't go back to the previous LRc version (I think the previous was 12 something) because I'm already using the new version's catalog for my work now. I do have a backup of the MIDI2LR settings. I can try a reboot, might be lucky with that. I'd forgetton how dependant on MIDI2LR I had become, going back to the sliders is so time consuming.

MIDI2LR 5.? was working fine with LRc before the LRc upgrade.

**Comment by Drounding** [comment]
Solved. 
I had pending windows updates. A re-boot and the new windows updates applied seems to have sorted the issue out.
